yt_dlp/extractor/cbc.py

```diff
@@ -473,49 +475,92 @@ def _real_extract(self, url):
 
 class CBCGemLiveIE(InfoExtractor):
     IE_NAME = 'gem.cbc.ca:live'
-    _VALID_URL = r'https?://gem\.cbc\.ca/live/(?P<id>\d+)'
-    _TEST = {
-        'url': 'https://gem.cbc.ca/live/920604739687',
-        'info_dict': {
-            'title': 'Ottawa',
-            'description': 'The live TV channel and local programming from Ottawa',
-            'thumbnail': 'https://thumbnails.cbc.ca/maven_legacy/thumbnails/CBC_OTT_VMS/Live_Channel_Static_Images/Ottawa_2880x1620.jpg',
-            'is_live': True,
-            'id': 'AyqZwxRqh8EH',
-            'ext': 'mp4',
-            'timestamp': 1492106160,
-            'upload_date': '20170413',
-            'uploader': 'CBCC-NEW',
+    _VALID_URL = r'https?://gem\.cbc\.ca/(?P<id>live(?:-event)?/(?P<key>\d+))'
```

Comment:
When changing `video_id` of an existing extractor `_old_archie_ids` must be implemented

Reply:
> Are there live and event with same id?

The prefix matters, at least in the URL, and it also returns a different JSON format. For now I don't use the `live` vs `live-event` to determine the JSON schema, but it could be useful in the future. In any case the number alone isn't enough to find the content.

> `_old_archive_ids` must be implemented.

None of the old videos are still running as they are livestreams. There are no examples of the old URL format available on the website. Is implementing this still necessary?

> id must not contain `/`

Will replace with `-`.

Reply:
> The prefix matters, at least in the URL, and it also returns a different JSON format. For now I don't use the `live` vs `live-event` to determine the JSON schema, but it could be useful in the future. In any case the number alone isn't enough to find the content.

You don't have to be able to retrieve the content back from the ID. It is only important that id+extractor is unique.

> None of the old videos are still running as they are livestreams. There are no examples of the old URL format available on the website. Is implementing this still necessary?

Fair enough. It's not needed then.

Reply:
> It is only important that id+extractor is unique for the video.

Okay, I guess I can't find repeats. Will fix.

Reply:
Fixed now.